pkg/apiserver/apiserver.go

```diff
@@ -61,7 +61,7 @@ func (s *APIServer) Run() {
 
 	file, err := os.Create(s.logFile)
 	if err != nil {
-		log.Fatalf(err.Error())
+		log.Fatalf("unable to create log file '%s': %s", s.logFile, err.Error())
```

Comment:
not needed RN, but I guess we want to start returning errors here instead of fataling as it will be called from "another" main